src/ssl.c

```diff
@@ -5424,8 +5422,16 @@ int wolfSSL_CertManagerEnableOCSPStapling(WOLFSSL_CERT_MANAGER* cm)
 
     return ret;
 }
-
-
+#if defined(SESSION_CERTS)
+STACK_OF(WOLFSSL_X509)* wolfSSL_get_peer_cert_chain(const WOLFSSL* ssl)
```

Comment:
If it's not possible to test against a known value in api.c, that is ok.

My question is more simple -- Are you 100% confident that this function works as expected?  We can't assume it's returning the right thing without at least double checking it locally in more detail.  If you have verified it locally against what OpenSSL returns for this function on your own computer during development, that is ok.